src/Components/ExternalResult/ResultList.tsx

```diff
@@ -254,7 +252,7 @@ export default function ResultList() {
 
   let resultList: any[] = [];
   if (data && data.length) {
-    resultList = data.map((result: any) => {
+    resultList = data.map((result: any, _idx: number) => {
```

Comment:
```suggestion
    resultList = data.map((result: any) => {
```

`_idx` is an optional parameter, so I think we can just avoid it when we don't use it, this keeps the code clean